src/@context/Pool/index.tsx

```diff
@@ -89,7 +92,7 @@ function PoolProvider({ children }: { children: ReactNode }): ReactElement {
       )
     }, refreshInterval)
     setFetchInterval(newInterval)
-  }, [fetchInterval, fetchAllData, accountId])
+  }, [fetchInterval, fetchAllData])
```

Comment:
did you notice the code comment above this method and made sure switching accounts correctly switches the interval too? Cause I had issues where I switched accounts, numbers switched correctly, but then old interval kicked in and fetched old numbers.

Reply:
i have to admit that i miss that, will investigate

Reply:
restored